core/src/main/java/org/apache/accumulo/core/metrics/MetricsUtil.java

```diff
@@ -121,6 +121,15 @@ public static List<Tag> getCommonTags() {
     return commonTags;
   }
 
+  // Centralize any specific string formatting for metric names and/or tags.
```

Comment:
This seems confusing. The micrometer convention is to use dots in the name and then have conversion done by the monitoring system (see https://micrometer.io/docs/concepts#_naming_meters)  Maybe just clarifying the javadoc for usage?  Otherwise, it seems counter to the micrometer convention and recommendations.

Reply:
I agree that micrometer wants dot separation and full lowercase strings. 

The dot removal was due to confusion about our current naming conventions as there are metric labels in MetricsProducer that use a underscore separator. I'm creating a different PR to sort that out. 

My intent with this method is to use it to normalize any internal accumulo object names that we use for labels to the micrometer standard.

Reply:
Switched the label names to use dot notation and follow micrometer standards. 
#3600 is open for the other metrics in MetricsProducer that use the `_` separator.